101-team-services-project-create/README.md

```diff
@@ -0,0 +1,9 @@
+# Deploy a new Team Services (VSTS) account and project.
+
+<a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2Fazure%2Fazure-quickstart-templates%2Fmaster%2F101-team-services-project-create%2Fazuredeploy.json" target="_blank">
+    <img src="http://azuredeploy.net/deploybutton.png"/>
+</a>
+
+This template allows you to deploy a new Team Services (VSTS) account and a new project.
```

Comment:
Makes sense @lorenpaulsen, thanks for the feedback. I took it into account with my last commit, let me know if you see anything else.